srcpkgs/v2ray/files/v2ray/run

```diff
@@ -1,4 +1,6 @@
 #!/bin/sh
 exec 2>&1
 
-exec chpst -u _v2ray v2ray run -config=/etc/v2ray/config.json 2>&1
+[ -r conf ] && . ./conf
+
+exec chpst -e /etc/v2ray/env -u _v2ray v2ray run ${OPTS:- -config=/etc/v2ray/config.json} 2>&1
```

Comment:
the conf file can already export environment variables. for consistency with other service please remove it.